github_api/comments.py

```diff
@@ -46,10 +46,7 @@ def get_reactions_for_comment(api, urn, comment_id):
     params = {"per_page": settings.DEFAULT_PAGINATION}
     reactions = []
 
-    try:
-        reactions = api("get", path, params=params)
-    except:
-        pass
+    reactions = api("get", path, params=params)
```

Comment:
If this fails, it's most likely due to a comment was deleted. If a comment was deleted, it shouldn't fail silently but rather should be handled